src/test/ui/async-await/partial-initialization-across-await.rs

```diff
@@ -0,0 +1,31 @@
+// compile-fail
```

Comment:
I believe this annotation is redundant.

-- Can you also add a short description of the test's purpose?